packages/components/cascader/src/index.vue

```diff
@@ -723,11 +720,7 @@ export default defineComponent({
         inputEl?.offsetHeight ||
         INPUT_HEIGHT_MAP[realSize.value] ||
         DEFAULT_INPUT_HEIGHT
-      addResizeListener(inputEl, updateStyle)
-    })
-
-    onBeforeUnmount(() => {
-      removeResizeListener(input.value?.$el, updateStyle)
+      useResizeObserver(inputEl, updateStyle)
```

Comment:
Move it outside of `onMounted`

Reply:
`inputEl` aka `input.value?.$el` is not a ref, so I don't think it's possible